src/helpers/config.ts

```diff
@@ -928,6 +954,7 @@ export const tokenParentCoins = {
   topethToken: 'topeth',
   polygonToken: 'polygon',
   tpolygonToken: 'tpolygon',
+  tavaxchainToken: 'tavaxchain'
```

Comment:
Why this is required?

Reply:
This is not required, was checking for options on WRW. Removed.